app/components/admin/participants/validation_tasks.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2 class="govuk-heading-m govuk-!-margin-top-9">Validation tasks</h2>
+
+<ol class="govuk-list govuk-list--number">
+  <% validation_steps.each do |step| %>
+    <li>
+      <table data-test="admin-participants-validation-tasks" class="govuk-table">
+        <thead  class="nhsuk-table__head">
+          <tr class="govuk-table__row">
+            <th class="govuk-table__header" colspan="2"><%= t :name, scope: "schools.participants.validations.#{profile.participant_type}.#{step}" %></td>
+          </tr>
+        </thead>
+        <tbody>
+          <tr class="govuk-table__row">
+            <td class="govuk-!-width-full">
```

Comment:
huh, what does this do in a responsive table?

Reply:
It forces this column to take as much space as possible, effectively pushing the tag as far to the right as possible. Tables are ultra-weird when it comes to sizing...